main.py

```diff
@@ -59,6 +61,26 @@ def get_city():
 
     return city.capitalize()
 
+def store_data():
+    db_handler = DBHandler('records_db.sqlite')
+
+    db_handler.create_table()# creates table if it does not exist
+
+#the following lines add data using dbhandler methods
+
+db_handler.add_travel_advisory(advisory_data)
```

Comment:
Should these lines be in a function? Where are the advisory_data, food_data, things_to_do_data and geo_location_data defined?

Reply:
They are defined in db_handler and data-presenter